test/loopback-dl.test.js

```diff
@@ -775,174 +775,177 @@ describe('Models attached to a dataSource', function() {
     Post.destroyAll(done);
   });
 
-  it('updateOrCreate should update the instance', function(done) {
-    Post.create({title: 'a', content: 'AAA'}, function(err, post) {
-      post.title = 'b';
-      Post.updateOrCreate(post, function(err, p) {
-        should.not.exist(err);
-        p.id.should.be.equal(post.id);
-        p.content.should.be.equal(post.content);
-        should.not.exist(p._id);
-
-        Post.findById(post.id, function(err, p) {
+  describe('updateOrCreate', function() {
+    it('updates instances', function(done) {
+      Post.create({title: 'a', content: 'AAA'}, function(err, post) {
+        post.title = 'b';
+        Post.updateOrCreate(post, function(err, p) {
+          should.not.exist(err);
           p.id.should.be.equal(post.id);
-          should.not.exist(p._id);
           p.content.should.be.equal(post.content);
-          p.title.should.be.equal('b');
+          should.not.exist(p._id);
```

Comment:
Telling `git` to ignore white space changes removes this artefact: https://github.com/strongloop/loopback-datasource-juggler/pull/1150/files?w=1

Unfortunately github does not allow comments in `w=1` mode :(